### ISSUE TITLE
Server Status - Error 500

### ISSUE THREAD

**Issue by ObadaS** [codalab/codabench#1508]
![image](https://github.com/codalab/codabench/assets/11889208/91e070e9-a60b-407c-a1dd-9c2cafe6f6ad)

```python
Internal Server Error: /server_status
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/views/generic/base.py", line 97, in dispatch
    return handler(request, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/views/generic/base.py", line 158, in get
    context = self.get_context_data(**kwargs)
  File "/app/src/apps/pages/views.py", line 92, in get_context_data
    submission.file_size = self.format_file_size(submission.data.file_size)
  File "/app/src/apps/pages/views.py", line 113, in format_file_size
    n = float(file_size)
TypeError: float() argument must be a string or a number, not 'NoneType'
Internal Server Error: /server_status
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/views/generic/base.py", line 97, in dispatch
    return handler(request, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/views/generic/base.py", line 158, in get
    context = self.get_context_data(**kwargs)
  File "/app/src/apps/pages/views.py", line 92, in get_context_data
    submission.file_size = self.format_file_size(submission.data.file_size)
  File "/app/src/apps/pages/views.py", line 113, in format_file_size
    n = float(file_size)
TypeError: float() argument must be a string or a number, not 'NoneType'
```
I tried restarting all the containers, the issue somehow persisted.

**Comment by ihsaan-ullah** [comment]
I think the problem is coming from the Value Error in except. It should be a generic exception or typeError

**Comment by ObadaS** [comment]
I am wondering why this problem seems to have appeared out of nowhere. And why it survived a restart.

**Comment by ObadaS** [comment]
The problem appears to have fixed itself for now, I can load the Server Status page again.

**Comment by ihsaan-ullah** [comment]
It is fixed because that submission with file size problem is 3 days old and we only show 2 days old submissions

**Comment by Didayolo** [comment]
I am still encountering this problem

**Comment by ObadaS** [comment]
@Didayolo It was fine this morning but came back a few hours ago. As @ihsaan-ullah said, as long as there is a submission with the problematic file size, it will cause this error.

**Comment by Didayolo** [comment]
Tested and merged: #1512
